### ISSUE TITLE
errors in APIv2 are swallowed and turned into 400 errors

### ISSUE THREAD

**Issue by valentijnscholten** [DefectDojo/django-DefectDojo#3679]
Since #3362 when an exception occurs during import, it gets swallowed. It's not logged anywhere and only a summary is returned in the API response:

```
{
    "detail": "Can't pickle <class 'tagulous.models.descriptors.TagRelatedManager'>: attribute lookup TagRelatedManager on tagulous.models.descriptors failed"
}
```

nothing in the logs so no traceback to find the root cause of the problem. 

it's caused by this construct in the api viewset: 

```
        try:
            serializer.save(push_to_jira=push_to_jira)
        except Exception as e:
            raise ParseError(detail=e)
```

**Comment by damiencarol** [comment]
@valentijnscholten it swallow the root exception?

**Comment by valentijnscholten** [comment]
Yes, and traceback etc.

**Comment by damiencarol** [comment]
merged in 1.12.0
